libshaderc_spvc/src/spvcir_test.cc

```diff
@@ -95,16 +95,16 @@ class SpvcIrParsingTest : public PassTest<::testing::Test> {
     )";
 
     before_ = {"OpCapability Shader", "OpCapability VulkanMemoryModelKHR",
-                  "OpExtension \"SPV_KHR_vulkan_memory_model\"",
-                  "OpMemoryModel Logical VulkanKHR"};
+               "OpExtension \"SPV_KHR_vulkan_memory_model\"",
+               "OpMemoryModel Logical VulkanKHR"};
 
     after_ = {"OpEntryPoint Vertex %1 \"shader\"",
-                   "%2 = OpTypeVoid",
-                   "%3 = OpTypeFunction %2",
-                   "%1 = OpFunction %2 None %3",
-                   "%4 = OpLabel",
-                   "OpReturn",
-                   "OpFunctionEnd"};
+              "%2 = OpTypeVoid",
+              "%3 = OpTypeFunction %2",
+              "%1 = OpFunction %2 None %3",
+              "%4 = OpLabel",
+              "OpReturn",
+              "OpFunctionEnd"};
```

Comment:
nit, the convention I have seen for spirv asm has been to have the opcode vertically alligned., so the '%4 = ' sticks out on the left.

If it is clang-format that is moving the lines around, you can use // clang-format off and // clang-format on before and after the region you don't want it to muck around with

Reply:
Thanks!